scripts/semconv/generate.sh

```diff
@@ -20,40 +21,41 @@ git fetch origin "$SPEC_VERSION"
 git reset --hard FETCH_HEAD
 cd ${SCRIPT_DIR}
 
+# stable attributes
 docker run --rm \
   -v ${SCRIPT_DIR}/semantic-conventions/model:/source \
   -v ${SCRIPT_DIR}/templates:/templates \
-  -v ${ROOT_DIR}/opentelemetry-semantic-conventions/src/opentelemetry/semconv/trace/:/output \
-  otel/semconvgen:$OTEL_SEMCONV_GEN_IMG_VERSION \
-  --only span,event,attribute_group \
+  -v ${ROOT_DIR}/opentelemetry-semantic-conventions/src/opentelemetry/semconv/:/output \
+  semconvgen:$OTEL_SEMCONV_GEN_IMG_VERSION \
   -f /source code \
   --template /templates/semantic_attributes.j2 \
-  --output /output/__init__.py \
-  -Dclass=SpanAttributes \
-  -DschemaUrl=$SCHEMA_URL
+  --output /output/_attributes.py \
+  --file-per-group root_namespace \
+  -Dfilter=is_stable
```

Comment:
So previously we were using (semconvgen) command line syntax with the `--only` flag to filter for attributes, how does `Dfilter` work in this case? Is `is_stable` defined somewhere or is it some sort of marker per attribute?

Reply:
Think about `filter` in `-Dfilter` as a string, it does not mean anything yet.
If you look into j2 files, you'd see something like

```jinja
{%- set filtered_attributes = attributes_and_templates | select(filter) | list %}
```

this is where `filter` is inserted as a sting.
Now it mean something.

`attributes_and_templates | select(filter)` filters attributes in the `attributes_and_templates` - it calls `filter` function on each attribute to test it.

There are built-in functions in Jinja, but `is_stable` `is_experimental` are custom defined in our codegen tooling - 
https://github.com/open-telemetry/build-tools/blob/feature/codegen-by-namespace/semantic-conventions/README.md#semanticattribute-operations